js/util/templateHelpers.js

```diff
@@ -36,6 +36,91 @@ define(["globals", "lib/faces", "lib/knockout", "util/helpers"], function (g, fa
         }
     };
 
+    ko.bindingHandlers.letterGrade = {
+      update: function(element, valueAccessor) {
+        var arg, output;
+        arg = ko.unwrap(valueAccessor());
+
+        if (arg < 0.20) {
```

Comment:
How do we know a priori that these cutoffs make sense? And that the same cutoffs can be used for all composite ratings?